src/main/java/seedu/address/logic/Messages.java

```diff
@@ -20,6 +20,8 @@ public class Messages {
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
     public static final String MESSAGE_EMPTY_CALL_HISTORY = "This person has no call history";
+    public static final String MESSAGE_USAGE_RESTRICTED_IN_HISTORY_VIEW = "This command is restricted in history view. "
```

Comment:
Maybe could mention that it's restricted because it uses INDEX? But this is kind of a personal pref so up to you

Reply:
I restrict the usage for both index and NRIC to prevent unintended changes. So, user must first verify exactly what they want to modify.

Reply:
Yep I understand that you restrict the command entirely, I was just thinking from the user's pov since they might be wondering why some commands (and not others) are restricted in history view. But once again, just a small nit.

Reply:
Ohh, so like adding the reason of restricting such commands here? I am thinking whether to add the reasons in ug or dg to clarify this, or adding it here is also ok but I wonder it will be a bit long.

Reply:
Yea adding in the ug/dg instead also sounds good! I just feel like we should probably mention it somewhere

Reply:
Yaa, I guess maybe I add the reason here also in case of they did not read the ug or dg.